Task10/Task10/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+namespace Task10
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите два натуральных числа от 5 до 20");
+            var m = int.Parse(Console.ReadLine());
+            var n = int.Parse(Console.ReadLine());
+            int[,] natNum = new int[m, n];
+            Random rand = new Random();
+
+            for (int i = 0; i < m; i++)
+            {
+                for (int j = 0; j < n; j++)
+                {
+                    int randomNum = rand.Next(0, 99);
+                    natNum[i, j] = randomNum;
+                }
+            }
+            PrintTable(natNum);
+
+            Console.Write("Введите натуральное число ");
+            var k = int.Parse(Console.ReadLine());
+            First(natNum, k);
+
+            Second(natNum);
+
+            Console.ReadKey();
+        }
+        static void PrintTable(int[,] arr)
+        {
+            for (int i = 0; i < arr.GetLength(0); i++)
+            {
+                for (int j = 0; j < arr.GetLength(1); j++)
+                    Console.Write($"{arr[i, j]} ");
+
+                Console.WriteLine();
+            }
+            Console.WriteLine();
+        }
+
+        static void First(int[,] arr, int k)
+        {
+            var arr1 = new int[] { k };
+            for (int i = 0; i < arr.GetLength(0); i++)
+            {
+                for (int j = 0; j < arr.GetLength(1); j++)
+                {
+                    Console.WriteLine(arr[i, j] > k ? $"{arr[i, j]} больше числа {k}" : $"В массиве нет элемента больше числа {k}");
+                    break;
+                }
+                break;
+            }
+        }
```

Comment:
Что-то не так с методом. Работает неверно. Листинг
```suggestion
78 34 27 73 28  8 54 68 83 90
55 85 13 18  2 17 67  1 15 29
58 31 17 75 78 56  1 26 16  8
94 34 74 86 54 55 30 85 50 49
 6 19 70 87 24 12 59  9 32 53
47 20 37 35 16 46 16 50 94 70

Введите натуральное число 89
В массиве нет элемента больше числа 89
```